src/lib/routes/admin-api/event.ts

```diff
@@ -45,23 +114,37 @@ export default class EventController extends Controller {
         } else {
             events = await this.eventService.getEvents();
         }
-        res.json({
+
+        const response: EventsSchema = {
             version,
-            events: this.fixEvents(events),
-        });
+            events: serializeDates(this.fixEvents(events)),
+        };
+        this.openApiService.respondWithValidation(
+            200,
+            res,
+            eventsSchema.$id,
+            response,
+        );
     }
 
     async getEventsForToggle(
-        req: Request<{ name: string }>,
-        res: Response,
+        req: Request<{ featureName: string }>,
```

Comment:
Was the parameter name wrong before? Will this break the API?

Reply:
Wasn't wrong, it just wasn't obvious what 'name' refers to. As far as I understand, it only affects what we call it internally and what it appears as in the open api UI. It's a path parameter, so no actual name required.

Reply:
Ah right, I was confusing it with a query string.